src/peft/tuners/lokr/layer.py

```diff
@@ -126,6 +126,22 @@ def reset_adapter_parameters_random(self, adapter_name: str):
         if adapter_name in self.lokr_t2:
             nn.init.kaiming_uniform_(self.lokr_t2[adapter_name], a=math.sqrt(5))
 
+    def reset_adapter_parameters_lycoris_way(self, adapter_name):
```

Comment:
Could you please add a short comment about the "lycoris way", as it may not be obvious to everyone.